common/scala/build.gradle

```diff
@@ -32,3 +32,7 @@ dependencies {
     compile 'org.apache.kafka:kafka-clients:0.10.0.0'
     compile 'org.apache.httpcomponents:httpclient:4.4.1'
 }
+
+tasks.withType(ScalaCompile) {
+    scalaCompileOptions.additionalParameters = ['-feature', '-deprecation', '-Xfatal-warnings', '-Ywarn-unused-import']
```

Comment:
if postfixOps is added, shall we nuke all the imports then?
@markusthoemmes will these compiler settings end up in the eclipse project setting also?